### PR TITLE
feat(cryptocom): watchOrderBook using updates and parseOrderBook for CountedOrderBook

### PR DESCRIPTION
**Pull request by pcriadoperez**
- Default to use SUBSCRIBE_AND_UPDATE to get higher frequency of updates
- Update parseOrderBook to also work for countedOrderBooks and IndexedOrderBooks
- Add `deepExtend` to allow passing parameters to topic params
- Update response from get_trades. It now passes timestamp in nanoseconds to optionaly if more accuracy is desired. In our case as timestamp is already set to ms, I ignored it.
- Add cryptocom static check

### CODE REVIEW

File: ts/src/test/static/response/cryptocom.json [hunk 1]
```diff
@@ -1809,6 +1809,3043 @@
...
+        ],
+        "httpResponse": {
+          "id": -1,
+          "method": "public/get-book",
+          "code": 0,
+          "result": {
+            "depth": 150,
+            "data": [
+              {
+                "bids": [
```
Comment (carlosmiei) [inline ts/src/test/static/response/cryptocom.json:1828]: @pcriadoperez static tests are good 😅 but we should try to keep them concise so they're easy to read/update/remove.

If the exchange response is huge, we can manually update  it and keep just 1 or 2 entry (you can see how I have done in methods like fetchPositions that sometimes return every single default position and ends up being thousands of lines

File: ts/src/pro/blockchaincom.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import blockchaincomRest from '../blockchaincom.js';
 import { NotSupported, AuthenticationError, ExchangeError } from '../base/errors.js';
 import { ArrayCache, ArrayCacheBySymbolById, ArrayCacheByTimestamp } from '../base/ws/Cache.js';
-import type { IndexType, Int, Str, OrderBook, Order, Trade, Ticker, OHLCV, Balances } from '../base/types.js';
```
Comment (carlosmiei) [inline ts/src/pro/blockchaincom.ts:6]: is this blockchaincom update on purpose?

File: ts/src/test/static/response/cryptocom.json (hunk 1 above)
Comment (pcriadoperez) [reply ts/src/test/static/response/cryptocom.json]: hehe good point. I reduced it to a test with a limit of 5

File: ts/src/pro/blockchaincom.ts (hunk 2 above)
Comment (pcriadoperez) [reply ts/src/pro/blockchaincom.ts:6]: Yes, with the change of parseOrderBook, I was able to clean up and delete `parseCountedOrderBook` from blockchaincom.  And after that IndexType is not used.

File: ts/src/test/static/response/cryptocom.json (hunk 1 above)
Comment (carlosmiei) [reply ts/src/test/static/response/cryptocom.json]: @pcriadoperez thanks! 🙂

**Pull request merged**